### DISCUSSION TITLE
2.0 Footage?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#629] [General]
The excitement I have for update 2.0 is welling up more and more as each day passes by. More than 9 months of painstakingly waiting for this update almost to an end. So many worries and anxieties about the update, like whether or not every single feature on the list will be in the update, or if some might get taken away during beta testing. Keiwan, if you're reading this, is there any chance we may see footage of the update before it comes out, like how FlipaClip teased a huge update in December with a trailer, but we're still waiting on said update?

**Comment by Rohitgit2412** [comment]
I think we need to control our excitement until the final release. Patience is a virtue, so let's wait and see. I have a feeling that releasing footage might be against beta testing protocols. Though i also want to have a sneak peek in what's coming for us but that shouldn't pressurize developer to rush and release an unstable or unfinished product. That would only lead to us criticizing it later. So, it's better if we're patient. ✌️
